lib/grape-swagger/doc_methods.rb

```diff
@@ -34,11 +34,28 @@ def setup(options)
         header['Access-Control-Allow-Origin']   = '*'
         header['Access-Control-Request-Method'] = '*'
 
+        namespaces = target_class.combined_namespaces
+        namespace_routes = target_class.combined_namespace_routes
+
+        namespace_routes_array = namespace_routes.keys.map do |local_route|
+          next if namespace_routes[local_route].map(&:route_hidden).all? { |value| value.respond_to?(:call) ? value.call : value }
+
+          original_namespace_name = target_class.combined_namespace_identifiers.key?(local_route) ? target_class.combined_namespace_identifiers[local_route] : local_route
```

Comment:
Hi Daniel, Thank you for the feedback.
https://github.com/ruby-grape/grape-swagger/pull/348
I have updated the code and the build is passing now.
looking forward for your feed back.